int64 imgID
int32 px
int32 py
float32 distance # estimated distance
---
geometry_msgs/Vector3 point